include/grpc++/server_credentials.h

```diff
@@ -64,6 +64,7 @@ struct SslServerCredentialsOptions {
   };
   grpc::string pem_root_certs;
   std::vector<PemKeyCertPair> pem_key_cert_pairs;
+  bool force_client_auth;
```

Comment:
Should we add a default ctor for this? Or update test/cpp/interop/server_helper.cc to set it there?